src/components/movieForm/index.tsx

```diff
@@ -112,33 +184,57 @@ const MovieForm: React.FC<IMovieFormProps> = ({
             data-testid='dropdown'
             onOptionSelect={handleOptionSelect}
             multiselect={true}
+            defaultSelectedOptions={initialMovieInfo?.genres ?? []}
             selectedOptions={selectedOptions}
```

Comment:
<img width="395" alt="image" src="https://user-images.githubusercontent.com/81073/235852602-74eae851-2146-41ac-a082-b9d8d103c3a6.png">
Option All shouldn't the in the list. It's not a Genre.